mve_intrinsics/mve.md

```diff
@@ -2,16 +2,22 @@
 title: Arm MVE Intrinsics
 version: 2021Q2
 date-of-issue: 02 July 2021
+# LaTeX specific variables
+landscape: true
```

Comment:
Given that we are using the `\newgeometry` command to set the list of intrinsic in landscape mode, I think we can start the document in portrait mode, and than switch to landscape mode only once we start the list of intrinsic. Same for the NEON specification.